docs/platforms/powershell/configuration/options.mdx

```diff
@@ -0,0 +1,185 @@
+---
+title: Basic Options
+description: "Learn more about how to configure the SDK. These options are set when the SDK is first initialized, passed to the init function as an object."
+sidebar_order: 1
+---
+
+SDKs are configurable using a variety of options. The options are largely standardized among SDKs, but there are some differences to better accommodate platform peculiarities. Options are set when the SDK is first
+initialized.
+
+<PlatformContent includePath="configuration/config-intro" />
+
+## Common Options
```

Comment:
Should this be an include file shared by .NET/Unity/PowerShell?

It's going out of sync quickly otherwise

Reply:
Same as above, this page is not .net specific, there are actually 18 copies of the same content right now, so any effort to reintegrate should be, IMO, done for all SDKs.